packages/lb-annotation/src/core/toolOperation/polygonOperation.ts

```diff
@@ -1424,21 +1446,25 @@ class PolygonOperation extends BasicToolOperation {
   }
 
   public onMouseUp(e: MouseEvent) {
-    if (this.isCombined) {
-      switch (e.button) {
-        case 0:
+    switch (e.button) {
+      case 0:
+        if (this.isCombined) {
           this.combine(e);
-          break;
-
-        case 2:
-          this.isCombined = false;
-          break;
-        default:
           return;
-      }
+        }
+        if (this.isSingleCropping) {
+          this.crop();
+          return;
+        }
+        break;
 
-      return;
+      case 2:
+        this.isCombined = false;
+        this.isSingleCropping = false;
```

Comment:
is isSingleCropping can implement in crop? Single and multiple should be handle in a function.